source/install.txt

```diff
@@ -1,177 +1,190 @@
 
-Compatibility
--------------
+.. tabs-pillstrip:: platforms
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+---------------------
 
-Software requirements to run |compass-short| are listed below:
+|compass-short| requires:
 
-.. include:: /includes/fact-compass-platforms.rst
+.. tabs-platforms::
 
-Installation
-------------
+   tabs:
+     - id: macos
+       content: |
 
-.. note::
+         - 64-bit version of macOS 10.10 or later
+         - MongoDB 3.0 or later.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+     - id: windows
+       content: |
 
-.. tabs::
+         - 64-bit version of Microsoft Windows 7 or later
+         - MongoDB 3.0 or later.
+         - `Microsoft .NET Framework version 4.5 or later <https://www.microsoft.com/en-us/download/details.aspx?id=30653>`__.
+           
+           The |compass-short| installer prompts you to install the
+           minimum required version of the .NET framework if it is not
+           already installed on your system.
 
-   tabs:
-     - id: osx
-       name: OS X
+     - id: debian
        content: |
 
-         Open the {+download-page+} and download the latest version of
-         |compass| for OS X. The package downloads as a ``.dmg`` disk
-         image. Once you have downloaded |compass-short|, click on the
-         ``.dmg`` file to open the disk image within the OS X Finder.
-         Drag the |compass| application to your
-         :guilabel:`Applications` folder and unmount the disk image.
+         - 64-bit version of Ubuntu 14.04 or later
+         - MongoDB 3.0 or later.
 
-         .. figure:: /images/compass/osx-install-dialog.png
-            :figwidth: 600px
+     - id: rhel
+       content: |
 
-         Launch |compass-short| by clicking on it from within the
-         :guilabel:`Applications` folder. Upon launch you will see the
-         following dialog:
+         - 64-bit version of RHEL 7+ or later
+         - MongoDB 3.0 or later.
 
-         .. figure:: /images/compass/application-downloaded-internet.png
-            :figwidth: 480px
+.. _macOS Terminal: https://support.apple.com/guide/terminal/welcome/mac
+.. _Windows Powershell: https://docs.microsoft.com/en-us/powershell/scripting/overview?view=powershell-6
+.. _zsh: https://www.zsh.org/
+.. _bash: https://www.gnu.org/software/bash/
+.. _ksh: http://www.kornshell.com/
 
-         Click Open to continue and launch |compass-short|.
+Download Compass
+----------------
 
-         .. note::
+To download |compass-short|, you can use your preferred web browser.
```

Comment:
I didn't quite understand your "stet" comment in your note in the GitHub review. Do we still need this sentence across each OS if we are asking users to open the downloads page by clicking on the link in step 1? (Presumably they are in a browser already). It would make things a bit more scannable / shorter.

Reply:
Stet is a proofreader's notation for [let it stand](https://en.wikipedia.org/wiki/Stet), i.e. leave it as is. We need something to clue in the reader as to what they are opening and visiting and how they do so. I am leaving it as is.

Reply:
Thank you.